.drone.yml

```diff
@@ -136,6 +136,8 @@ steps:
       from_secret: FIREBASE_NAME
     FIREBASE_SECRET:
       from_secret: FIREBASE_SECRET
+    FIREBASE_SHARED_SECRET:
+      from_secret: FIREBASE_SHARED_SECRET
```

Comment:
Note that we're only enabling this for the container that runs UI tests, and not the one that runs unit tests.